docs/releases/adding-new-firmwares.md

```diff
@@ -10,32 +10,25 @@ Package `@trezor/connect-common` is a public NPM package used as dependency of `
 
 1. Complete the firmware release process including firmware signing.
 
-2. Add firmwares to `packages/connect-common/files/firmware/*` and modify its `releases.json` file. See [90bb548](https://github.com/trezor/trezor-suite/commit/90bb548aec06c9b4816c9a87b2ffa5fcade99f29) for example.
+1. Add firmwares to `packages/connect-common/files/firmware/*` and modify its `releases.json` file. See [90bb548](https://github.com/trezor/trezor-suite/commit/90bb548aec06c9b4816c9a87b2ffa5fcade99f29) for example.
 
     - To retrieve `fingerprint`, run `trezorctl firmware-update -f {path-to-the-bin}` for each binary separately (you don't have to confirm the update on device unless you want to). Look for `Firmware fingerprint:` row.
 
-3. Remove older binaries so they are not bundled in desktop app any more, but always keep:
+1. Remove older binaries so they are not bundled in desktop app any more, but always keep:
 
     - the intermediary FW for Trezor One [packages/connect-common/files/firmware/1/trezor-inter-1.10.0.bin](https://github.com/trezor/trezor-suite/blob/develop/packages/connect-common/files/firmware/1/trezor-inter-1.10.0.bin)
     - and 2.1.1 for model T [packages/connect-common/files/firmware/2/trezor-2.1.1.bin](https://github.com/trezor/trezor-suite/blob/develop/packages/connect-common/files/firmware/2/trezor-2.1.1.bin)
 
     See [#4262](https://github.com/trezor/trezor-suite/issues/4262) for explanation.
 
-4. Release new version of `connect-common` package.
+1. Release new version of `connect-common` package.
```

Comment:
to discuss:
connect-common doesn't have to be released before suite freeze, it needs to be released to propagate changes to connect.trezor.io tho.
I think freeze could be done without updating version

Reply:
I thought so too, but we need to use the new version of `connect-common` in `trezor-connect` so the new firmwares are bundled in the freezed Suite - otherwise it's not possible to test the update to new FW version during freeze. Or is there any better way? Maybe we are not using the `connect-iframe` properly?

Reply:
theoretically if you dont update `connect-common` version then new files should be available out of the box